frontend/www/js/omegaup/components/arena/ContestListv2.vue

```diff
@@ -208,6 +305,94 @@ export default class ArenaContestList extends Vue {
   startContestDate(contest: types.ContestListItem): string {
     return contest.start_time.toLocaleDateString();
   }
+
+  orderByTitle() {
+    this.currentOrder = ContestOrder.Title;
+    this.sortedContestList;
+  }
+
+  orderByEnds() {
+    this.currentOrder = ContestOrder.Ends;
+    this.sortedContestList;
+  }
+
+  orderByDuration() {
+    this.currentOrder = ContestOrder.Duration;
+    this.sortedContestList;
+  }
+
+  orderByOrganizer() {
+    this.currentOrder = ContestOrder.Organizer;
+    this.sortedContestList;
+  }
+
+  orderByContestants() {
+    this.currentOrder = ContestOrder.Contestants;
+    this.sortedContestList;
+  }
+
+  orderBySignedUp() {
+    this.currentOrder = ContestOrder.SignedUp;
+    this.sortedContestList;
+  }
+
+  get sortedContestList(): types.ContestListItem[] {
+    function compareNumber(a: number, b: number): number {
+      if (a < b) {
+        return 1;
+      } else if (a > b) {
+        return -1;
+      }
+      return 0;
+    }
+    let sortBy: (a: types.ContestListItem, b: types.ContestListItem) => number;
+    switch (this.currentOrder) {
+      case ContestOrder.Title:
+        sortBy = (a, b) => a.title.localeCompare(b.title);
+        break;
+      case ContestOrder.Ends:
+        sortBy = (a, b) =>
+          compareNumber(a.finish_time.getDate(), b.finish_time.getDate());
+        break;
+      case ContestOrder.Duration:
+        sortBy = (a, b) =>
+          compareNumber(
+            a.finish_time.getTime() - a.start_time.getTime(),
+            b.finish_time.getTime() - a.start_time.getTime(),
```

Comment:
```suggestion
            b.finish_time.getTime() - b.start_time.getTime(),
```
Aquí también eso estaba mal y lo cambié